apps/web/src/features/shared/scroll-to-top/_index.scss

```
@@ -22,12 +22,22 @@
     width: 48px;
     height: 48px;
     bottom: calc(env(safe-area-inset-bottom) + 4.75rem);
+    // Match the FAB's `transition-transform duration-300` (Tailwind easing) so
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)